manifest/com.Sinduy/QuestPro4Resonite/info.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "QuestPro4Resonite",
+	"id": "com.Sinduy.QuestPro4Resonite",
+	"description": "With Quest Pro, you can eye tracking and facial tracking whih ALXR also You need read the instruction in the github page to install the mod.",
+	"category": "Hardware Integration",
+	"sourceLocation": "https://github.com/sjsanjsrh/QuestPro4Resonite",
+	"additionalAuthors": [
+		"dfgHiatus",
+		"Geenz",
+		"Scar Vorrix",
+		"Dante Tucker"
+	],
+	"versions": {
+		"2.1.3": {
+			"releaseUrl": "https://github.com/sjsanjsrh/QuestPro4Resonite/releases/tag/2.1.3",
+			"dependencies": {
+				"com.Sinduy.libalxr-sharp": {
+					"version": "^1.2.0"
+				}
+			},
+			"artifacts": [{
+				"url": "https://github.com/sjsanjsrh/QuestPro4Resonite/releases/download/2.1.3/QuestProModule.dll",
```

Comment:
[assembly: AssemblyFileVersion("1.0.0.0")] and [assembly: AssemblyVersion("1.0.0.0")] don't match the version defined in the mod https://github.com/resonite-modding-group/resonite-mod-manifest/wiki/Submission-Guidelines#versioning

Reply:
Non-blocking note that libalxr-sharp also doesn't set these